--- AliasController.php.old	2022-07-20 20:37:24.159324000 +0200
+++ AliasController.php.new	2022-07-20 19:29:58.334986000 +0200
@@ -327,10 +327,10 @@
             $this->notify( 'alias', 'toggleActive', 'preflush', $this, [ 'active' => $this->getAlias()->getActive() ] );
             $this->getD2EM()->flush();
             $this->notify( 'alias', 'toggleActive', 'postflush', $this, [ 'active' => $this->getAlias()->getActive() ] );
+            print 'ok';
         } else {
             print 'ko';
         }
-        print 'ok';
     }
 
 
