--- MailboxController.php.old	2022-07-20 20:37:24.163325000 +0200
+++ MailboxController.php.new	2022-07-20 20:38:29.140738000 +0200
@@ -408,18 +408,22 @@
         if( !$this->getMailbox() )
             print 'ko';
 
-        $this->getMailbox()->setActive( !$this->getMailbox()->getActive() );
-        $this->getMailbox()->setModified( new \DateTime() );
+        if($this->notify( 'mailbox', 'toggleActive', 'preToggle', $this, [ 'active' => $this->getMailbox()->getActive() ]) === true) {
+            $this->getMailbox()->setActive( !$this->getMailbox()->getActive() );
+            $this->getMailbox()->setModified( new \DateTime() );
 
-        $this->log(
-            $this->getMailbox()->getActive() ? \Entities\Log::ACTION_MAILBOX_ACTIVATE : \Entities\Log::ACTION_MAILBOX_DEACTIVATE,
-            "{$this->getAdmin()->getFormattedName()} " . ( $this->getMailbox()->getActive() ? 'activated' : 'deactivated' ) . " mailbox {$this->getMailbox()->getUsername()}"
-        );
+            $this->log(
+                $this->getMailbox()->getActive() ? \Entities\Log::ACTION_MAILBOX_ACTIVATE : \Entities\Log::ACTION_MAILBOX_DEACTIVATE,
+                "{$this->getAdmin()->getFormattedName()} " . ( $this->getMailbox()->getActive() ? 'activated' : 'deactivated' ) . " mailbox {$this->getMailbox()->getUsername()}"
+            );
 
-        $this->notify( 'mailbox', 'toggleActive', 'postflush', $this, [ 'active' => $this->getMailbox()->getActive() ] );
-        $this->getD2EM()->flush();
-        $this->notify( 'mailbox', 'toggleActive', 'postflush', $this, [ 'active' => $this->getMailbox()->getActive() ] );
-        print 'ok';
+            $this->notify( 'mailbox', 'toggleActive', 'preflush', $this, [ 'active' => $this->getMailbox()->getActive() ] );
+            $this->getD2EM()->flush();
+            $this->notify( 'mailbox', 'toggleActive', 'postflush', $this, [ 'active' => $this->getMailbox()->getActive() ] );
+            print 'ok';
+        } else {
+            print 'ko';
+        }
     }
 
 
